### PR TITLE
Index HTML made Responsive with carousel

### PR DESCRIPTION
**Pull request by vbvh98**
Carousel loads mobile posters when on tablet or mobile device and desktop images on desktops.
page made responsive for smaller screen sizes.

### CODE REVIEW

**Comment by mooocer** [comment]
The carousel buttons sometimes go inactive such that I keep clicking and it doesn't work.

File: index.html [hunk 1]
```diff
@@ -139,153 +113,124 @@
...
+                    <div class="col-sm-4 text-right mr-0 pr-0 ml-0 pl-0" style="width: 100%; overflow-x: hidden;">
+                        <img height="448px" src="img/about.jpg" alt="Image" />
+                    </div>
                 </div>
             </div>
         </div>
-    </div>
-    <!-- End Culture -->
-
-    <!--========== Contact Us ==========-->
```
Comment (harshitlikhar) [inline index.html]: Why is Contact Us Section and its CSS classes is Deleted?

File: index.html [hunk 2]
```diff
@@ -139,153 +113,124 @@
...
-            <h3 class="h2">Contact Us</h3>
-            <p class="para_cu"  >For any queries.</p>
-        </div>
-
-        <div style="text-align: center;">
-            <a id="cu_a"  href="mailto:hacksocindore@gmail.com" target="_top"  class=" btn btn-secondary btn-lg" >SEND US AN EMAIL!</a>
-        </div>
-    </div>
-
-    <!--========== Contact Us  End==========-->
```
Comment (keshavlikhar) [inline index.html:239]: Why is Contact Us section and its classes are deleted?

Comment (vbvh98) [reply index.html:239]: will fix it in a while

**Comment by mooocer** [comment]
@vbhv98 status?

**Pull request closed without merging**